openfisca_nsw_community_gaming/variables/proceeds_prizes_and_fees.py

```diff
@@ -95,9 +95,9 @@ class value_of_individual_prize(Variable):
 class no_fees_charged_for_conducting_game(Variable):
     value_type = bool
     entity = Organisation
-    label = """Other than a prize, is there any salary, wage, fee, commission,
-        percentage or other benefit given, paid to or taken by a person in
-        connection with the conduct of the gaming activity?"""
+    label = """Will any entry or other fee be charged to participate
+    in the gaming activity? (other than purchasing goods or services
+    at a normal retail price?)"""
```

Comment:
There shouldn't really be a question mark after price. Perhaps the parenthesis should go before the question mark after activity? It's kind of a funny fragment.